source/offline_data.template.h

```diff
@@ -620,6 +634,13 @@ namespace ryujin
           }   /* for j */
         }     /* for q */
       }
+
+      /*
+       * Compute block inverse of mass matrix:
+       */
+
+      // FIXME: rewrite with CellwiseInverseMassMatrix
+      cell_mass_matrix_inverse.invert(cell_mass_matrix);
```

Comment:
I agree on this part here; the question is if we can get around with the cell-wise inverse action on a vector (as implemented by that class) in the hyperbolic module or if we really need to store the ij entries of the inverse matrix. In the latter case, it would also be preferable to store a collection of dense matrices in a singular contiguous array of memory, rather than doing indirect addressing into a generic CRS object, but that is for a later stage of code optimization.)